features/mbedtls/importer/Makefile

```diff
@@ -62,6 +71,23 @@ rsync:
 	cp $(MBED_TLS_DIR)/LICENSE $(TARGET_PREFIX)
 	cp $(MBED_TLS_DIR)/apache-2.0.txt $(TARGET_PREFIX)
 	#
+	# Create Mbed Crypto target folder
+	mkdir -p $(TARGET_PREFIX_CRYPTO)
+	#
+	# Copying Mbed Crypto into Mbed OS..
+	rm -rf $(TARGET_SRV_IMPL)
+	rm -rf $(TARGET_SPE)
+	rm -rf $(TARGET_PSA_ACCELERATION)
+
+	mkdir -p $(TARGET_SRV_IMPL)
```

Comment:
Why are we removing and making these directories? Unlike the other directories being made, won't there be other files in these directories which aren't being imported?

Reply:
I think only `$(TARGET_PSA_ACCELERATION)` shouldn't be removed, the others are created and only the importer copies files into them.